algosdk/error.py

```diff
@@ -135,6 +135,11 @@ def __init__(self):
             "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ")
 
 
+class KeyregOnlineTxnInitError(Exception):
```

Comment:
@jasonpaulos I added this error to check that all the attributes are not `None`.